src/components/Forecast/index.tsx

```diff
@@ -65,13 +70,19 @@ function Forecast({ data }: Props) {
       )}
 
       {data.map((bud) => (
-        <S.EachDay key={bud.day}>
-          <S.BigFont>{bud.day}</S.BigFont>
-          <S.SmallFont>{bud.date}</S.SmallFont>
-          <S.Images src={bud.weather} alt="weather" />
+        <S.EachDay key={bud.time}>
+          <S.BigFont>{bud.date}</S.BigFont>
+          <S.SmallFont>{bud.time}</S.SmallFont>
+          {bud.weather === "Clouds" && <S.Images src={clouds} alt="weather" />}
+          {bud.weather === "Clear" && <S.Images src={sun} alt="weather" />}
+          {bud.weather === "Rain" && (
+            <S.Images src={cloudwithrain} alt="weather" />
+          )}
```

Comment:
You can refactor this by creating something like `getWeatherIcon` and there choose clouds, sun or something else. And leave here only `<S.Images src={getWeatherIcon(bud.weather)} alt="weather" />`